app/vmalert/README.md

```diff
@@ -119,6 +119,12 @@ name: <string>
 # `eval_offset` can't be bigger than `interval`.
 [ eval_offset: <duration> ]
 
+# Optional
+# Adjust the `time` parameter of group evaluation requests to match intentional query delay from datasource.
```

Comment:
I think we should do better explaining this. Remember, user should receive enough of understanding from this decription to decide whether to change this param.

Reply:
Updated, plz check.